packages/core/src/components/autosuggest/autosuggest.tsx

```diff
@@ -537,11 +548,12 @@ export class Autosuggest {
                   (this.notFound && (
                     <li>
                       <span class="value">
-                        {!this.notFoundLabel ? (
-                          this.showInputValueNotFound(`${this.inputValue} is niet gevonden.`)
-                        ) : (
-                          <span>{this.notFoundLabel}</span>
-                        )}
+                        {this.notFoundLabel ||
+                          this.showInputValueNotFound(
+                            this.i18nInstance
+                              ? this.i18nInstance.t("notFound", { inputValue: this.inputValue })
+                              : `${this.inputValue} is niet gevonden`,
```

Comment:
Hier staat nog steeds een taal hardcoded, terwijl we alle teksten naar de i18n files willen hebben

Reply:
Gedaan